dusty/commands/env.py

```diff
@@ -0,0 +1,94 @@
+import functools
+import os
+
+from prettytable import PrettyTable
+
+from ..log import log_to_client
+from ..compiler.spec_assembler import get_assembled_specs, get_specs
+from ..config import get_env_config, save_config_value
+from .. import constants
+from ..payload import daemon_command
+
+def _save_env(global_env_dict):
+    save_config_value(constants.CONFIG_ENV_KEY, global_env_dict)
```

Comment:
Never use `global` in a variable name unless it's actually a global variable, which this isn't